src/workspace.coffee

```diff
@@ -550,7 +556,7 @@ class Workspace extends Model
   # Extended: Create a new text editor.
   #
   # Returns a {TextEditor}.
-  buildTextEditor: (params) ->
+  buildTextEditor_: (params) ->
```

Comment:
The problem I found was that if `@buildTextEditor` was called, which then called `realThis.buildTextEditor`, we'd infinitely recurse.